controller/array_action/array_mediator_ds8k.py

```diff
@@ -23,7 +23,10 @@
 
 # response error codes
 ERROR_CODE_INVALID_CREDENTIALS = 'BE7A002D'
+KNOWN_ERROR_CODE_INVALID_CREDENTIALS = 'BE7A0029'
```

Comment:
why `KNOWN_`?

Reply:
I dont have the spec for the Errors and I dont know that error (same name without the known), so I didn't remove it.

Reply:
I think it's true for all error codes here, and I think they should be aligned, so I would remove the prefix

Reply:
if you want them to have different names, I would use `_1` + `_2` suffixes, or simply a list of them